sqlup-mode.el

```diff
@@ -67,19 +67,12 @@
 this mode's logic will be evaluated.")
 
 (defconst sqlup-eval-keywords
-  '((postgres "EXECUTE"))
-"List of keywords introducing eval strings, organised by dialect.")
+  '((postgres "EXECUTE" "format("))
```

Comment:
Why is this now two strings instead of just one? Can `EXECUTE` and `format(` be used separately?